src/Connection.ts

```diff
@@ -16,18 +22,18 @@ export interface MomoConnectionOptions {
 
 export class Connection {
   private constructor(
-    private readonly mongoClient: MongoClient,
+    public readonly mongoClient: MongoClient,
```

Comment:
This probably does not need to be `public`, right?

Reply:
Good catch, thank you! I accidentally commited this. It is a relic of my attempts to write an integration test for this